api/src/main/java/org/apache/apex/api/ApexPlugin.java

```diff
@@ -18,12 +18,15 @@
  */
 package org.apache.apex.api;
 
+import org.apache.hadoop.classification.InterfaceStability;
+
 import com.datatorrent.api.Component;
 
 /**
  * Marker interface for ApexPlugins.
  * @param <T>
  */
+@InterfaceStability.Evolving
 public interface ApexPlugin<T extends ApexPluginContext> extends Component<T>
```

Comment:
There are two plugins DAGSetupPlugin and DAGExecutionPlugin. ApexPlugin is kept as a common parent of these as per @PramodSSImmaneni request during review. @PramodSSImmaneni  an you and @vrozov come on conclusion on the naming and package. I will do the changes accordingly.